lib/bolt/util/format.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Bolt
+  module Util
+    module Format
```

Comment:
Not really attached to having smaller utility modules like this. Thought it'd be nice just from an organizational standpoint (for example we could have `Bolt::Util::Files` as well), but I'm also fine with keeping everything in `Bolt::Util`.

Reply:
I think the word `util` is a code smell, which means having a single monolithic `util` namespace is especially a code smell. So breaking it up seems like an improvement.

Reply:
Good point. I found [this](https://www.jmgundersen.net/blog/utility-and-helper-classes-a-code-smell) blog post which argues the same, and I think they make some very valid points as well.

Reply:
Breaking this out is definitely great, and like...there are other ways to make these functions available in Puppet plan functions but to me a utility class seems fine.